src/questionnaire.js

```diff
@@ -56,7 +56,7 @@ module.exports = [
   {
     name: "theme",
     type: "list",
-    message: "Do you want Dark Mode?",
+    message: "Do you want ThemeProvider?",
```

Comment:
@manjeetbeniwalinnostax 
I think, we need to gave different theme options to user.

Reply:
{name: "Light/Dark Mode", value: "light-dark-mode"}
{name: "None", value: false}